Doppler.Jobs.Server/appsettings.json

```diff
@@ -10,9 +10,21 @@
   "AllowedHosts": "*",
   "Jobs": {
     "WorkerServiceJob": {
-      "IntervalCronExpression": "0/10 * * * * ?",
+      "IntervalCronExpression": "0 18 * * mon-fri",
```

Comment:
```
# ┌───────────── minute (0 - 59)
# │ ┌───────────── hour (0 - 23)
# │ │ ┌───────────── day of the month (1 - 31)
# │ │ │ ┌───────────── month (1 - 12)
# │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
# │ │ │ │ │                                   7 is also Sunday on some systems)
# │ │ │ │ │
# │ │ │ │ │
# * * * * * command to execute
```

Does it work at 18hs ARG or UTC? Is it fine using 18 hours for all countries?

Reply:
Oh, I understand: https://github.com/MakingSense/doppler-jobs/commit/3bf4b10bdadf4513809a8e4f691d94481449b369#diff-c48344bd07d6708e5b0d1d02e73992b7R17

It is from ARG.